src/questions/decentralized_exchanges.yaml

```diff
@@ -0,0 +1,786 @@
+      key: B
+    - content: It also shows the prices and volumes that people in the market are willing to buy and sell the security for.
+      key: C
+    - content: It does not show the prices and volumes that people in the market are willing to buy and sell the security for.
+      key: D
+- uuid: ad432625-2565-4976-ad0c-20c24a1aeae9
+  type: SingleChoice
+  content: What are the types of CEX based on the order book model storage?
+  hint: Centralized Database and Blockchain Database
+  explanation: There are two types of Order Book Model:- 1) Stores all data on a centralized database & 2) Stores all data on distributed, decentralized, and immutable data blocks of a Blockchain database.
+  answerKeys:
+    - C
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Low
+  choices:
+    - content: CEX with Server Order Book Model
+      key: A
+    - content: CEX with Onchain Order Book Model
+      key: B
+    - content: Both of these
+      key: C
+    - content: None of these
+      key: D
+- uuid: ce272bb4-0b0a-404d-a816-f64706893a04
+  type: MultipleChoice
+  content: CEX with Server Order Book Model has
+  hint: Faster, Uncensored, Free cancelation
+  explanation: Due to centralized governance they are faster as no validator/miner action is required in trade execution, and not to pay any gas fees for transactions hence, no fees are required for canceled orders, also all types of traders’ data are in the hand of a central authority so no censorship resistance.
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Medium
+  choices:
+    - content: Faster trade matching and settlement
+      key: A
+    - content: Slow trade matching and settlement
+      key: B
+    - content: No fees for cancelled orders
+      key: C
+    - content: No censorship resistance
+      key: D
+- uuid: 4985ad13-5ce6-4d04-8d66-96b378c13438
+  type: MultipleChoice
+  content: CEX with Onchain Order Book Model has
+  hint: Slower, Censored
+  explanation: Due to the Decentralized and Distributed Ledger storage system they are slower as the validator/miner action is required in trade execution and have to pay gas fees for transactions hence, fees are required for canceled orders, also all types of traders’ data are censorship resistance.
+  answerKeys:
+    - A
+    - D
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Medium
+  choices:
+    - content: Slow trade matching and settlement
+      key: A
+    - content: No fees for cancelled orders
+      key: B
+    - content: No Censorship resistance
+      key: C
+    - content: Censorship resistance
+      key: D
+- uuid: 39cd2b05-81f4-4b0a-a540-e1d21fa275b7
+  type: SingleChoice
+  content: ____ are basic sets of rules that allow data and cryptocurrencies to be shared securely between different nodes (i.e computers) in the blockchain network.
```

Comment:
This is a confusing question as `Liquidity Pools`, `Order Books` and `Exchanges` are protocols too